### PR TITLE
Csv generation

### PR DESCRIPTION
**Pull request by FadySalama**
Fix a bug when generating assertion.csv out of a single file, add needReview.csv generation, add comments column

### CODE REVIEW

File: packages/cli/index.js
Comment (egekorkan) [inline packages/cli/index.js:233]: what is happening here? Can you comment

File: packages/assertions/assertions-csv/manual.csv
Comment (egekorkan) [inline packages/assertions/assertions-csv/manual.csv:1]: Is the diff this big due to addition of comment field and its associated comma?

**Review by egekorkan — CHANGES_REQUESTED** [review]
Some minor comments. Also, we desperately need a readme on what is happening here since I have already asked but forgot the real details.

Also, I have merged #327 so you need to rerun the manual csv generation :)

File: packages/assertions/assertions-csv/manual.csv
Comment (FadySalama) [reply packages/assertions/assertions-csv/manual.csv:1]: Yes, that's really it

File: packages/cli/index.js
Comment (FadySalama) [reply packages/cli/index.js:233]: Basically what is happening is the outReport function takes the `input (-i)` parameter from the CLI and uses that as part of the `id` in the `output` path. The problem happens when you specify a path for both the `input (-i)` and the `output (-o)`.  When you do this, both get concatenated together and produce an error. I made sure that from the input path we basically take only the name of the file as for the `id` in the output path

Comment (egekorkan) [reply packages/cli/index.js:233]: I see, that is good. Could you add this as a comment

Comment (FadySalama) [reply packages/cli/index.js]: I found where the problem was, it wasn't in this line, so I reverted that line to its former state

Comment (FadySalama) [reply packages/cli/index.js]: I found where the problem truly is. I fixed that and now it should work

**Comment by egekorkan** [comment]
I think it is ready to merge after getting a readme :)

**Comment by FadySalama** [comment]
> I think it is ready to merge after getting a readme :)

A readme to what exactly?

**Comment by egekorkan** [comment]
> > I think it is ready to merge after getting a readme :)
> 
> A readme to what exactly?

Under `assertions-csv` folder that explains why this exists and how it works

**Comment by FadySalama** [comment]
> > > I think it is ready to merge after getting a readme :)
> > 
> > 
> > A readme to what exactly?
> 
> Under `assertions-csv` folder that explains why this exists and how it works

@egekorkan README has been added, awaiting your approval 😃

**Comment by egekorkan** [comment]
Looking good. Gonna wait for CI and merge

**Pull request merged**
